pulseguardian/guardian.py

```diff
@@ -5,19 +5,28 @@
 import logging
 import re
 import time
+import requests
 
 from pulseguardian import config
 from pulseguardian.logs import setup_logging
 from pulseguardian.management import PulseManagementAPI
 from pulseguardian.model.base import init_db, db_session
-from pulseguardian.model.user import PulseUser
+from pulseguardian.model.user import PulseUser, User
 from pulseguardian.model.queue import Queue
-from pulseguardian.sendemail import sendemail
+from pulseguardian.sendemail import sendemail as email
 
 logging.getLogger("requests").setLevel(logging.WARNING)
 
 setup_logging(config.guardian_log_path)
 
+def sendemail(to_addrs=[], subject=None, text_data=None, from_addr=config.email_from,
```

Comment:
Also don't use mutable objects as default parameters (see, for example, http://www.dskims.com/least-astonishment-in-python-the-mutable-default-argument/). Instead, you should use something like `None` and then check for that value in the function body and initialize it to `[]`.

Although, in this case I don't think a default value is necessary. It doesn't make much sends to try to send an email with no recipients. :)